jekyll/_cci2/config-policy-management-overview.adoc

```diff
@@ -259,12 +259,16 @@ When a policy is evaluated in the context of a triggered pipeline the following
 
 [source,shell]
 ----
-project_id    (CircleCI Project UUID)
-branch        (string)
-build_number  (number)
+project_id                   (CircleCI Project UUID)
+build_number                 (number)
+
+vcs.branch                   (string)
+vcs.release_tag              (string)
+vcs.origin_repository_url    (string)
+vcs.target_repository_url    (string) # URL of the target repo (in a pull request from a forked repo)
```

Comment:
I don't know that the comment is helpful. I would rather we have a small section below explaining the difference between the origin and target repositories in a 1 or 2 concise sentences.

Reply:
@davidmdm @kbatuigas 
How about we just provide a reference to: https://circleci.com/docs/webhooks-reference/#vcs
This page already has descriptions of all the metadata fields we get, vcs or otherwise.